activerecord/test/cases/store_test.rb

```diff
@@ -174,9 +216,9 @@ class StoreTest < ActiveRecord::TestCase
       store_accessor :data, :area, :volume
     end
 
-    assert_equal [:color], first_model.stored_attributes[:data]
-    assert_equal [:color, :width, :height], second_model.stored_attributes[:data]
-    assert_equal [:color, :area, :volume], third_model.stored_attributes[:data]
+    assert_equal ['color'], first_model.stored_attributes[:data]
+    assert_equal ['color', 'width', 'height'], second_model.stored_attributes[:data]
+    assert_equal ['color', 'area', 'volume'], third_model.stored_attributes[:data]
```

Comment:
People could potentially be relying on the symbol return. Let's keep 'em that way. Hell changing the type might break the underlying serialize mechanism.